web-common/src/features/file-explorer/utils.ts

```diff
@@ -0,0 +1,14 @@
+import { removeLeadingSlash } from "@rilldata/web-common/features/entity-management/entity-mappers";
+
+export function isCurrentActivePage(
+  filePath: string,
+  currentFile: string | undefined,
+  isDir: boolean,
+) {
+  filePath = removeLeadingSlash(filePath);
+  currentFile = removeLeadingSlash(currentFile ?? "");
+  return (
+    currentFile !== "" &&
```

Comment:
Adding a guard clause at the top of the function could be better:
```
if (currentFile === undefined) return false
```

Further, I'd wonder if the function even needs to accept an `undefined` `currentFile`.

Reply:
`$page.params.file` could be undefined when user is on home. Moving that check here for now. Felt cleaner to deal with `$page` here itself.